docs/source/markdown/guides/how_to/data/custom_data.md

```diff
@@ -0,0 +1,372 @@
+- Classification with normal and abnormal images, and
+- Classification with only normal images.
+
+### With Normal and Abnormal Images
+
+We could use [Folder](../../reference/data/image/folder.md) datamodule to train
+a model on this dataset. We could run the following python code to create the
+custom datamodule:
+
+:::::{dropdown} Code Syntax
+:icon: code
+
+::::{tab-set}
+:::{tab-item} API
+:sync: label-1
+
+```{literalinclude} ../../../../snippets/data/image/folder/classification/default.txt
+:language: python
+```
+
+```{note}
+As can be seen above, we only need to specify the ``task`` argument to ``classification``. We could have also use ``TaskType.CLASSIFICATION`` instead of ``classification``.
+```
+
+The [Folder](../../reference/data/image/folder.md) datamodule will create training, validation, test and
+prediction datasets and dataloaders for us. We can access the datasets
+and dataloaders by following the same approach as in the segmentation
+task.
+
+When we check the samples from the dataloaders, we will see that the
+`mask` key is not present in the samples. This is because we do not need
+the masks for the classification task.
+
+```{literalinclude} ../../../../snippets/data/image/folder/classification/dataloader_values.txt
+:language: python
+```
+
+Training the model is as simple as running the following command:
+
+```{literalinclude} ../../../../snippets/train/api/classification/model_and_engine.txt
+:language: python
+```
+
+where we train a Patchcore model on this custom dataset with default model parameters.
+
+:::
+
+:::{tab-item} CLI
+:sync: label-2
+
+Here is the dataset config to create the same custom datamodule:
+
+```{literalinclude} ../../../../snippets/config/data/image/folder/classification/cli/default.yaml
+:language: yaml
+```
+
+Assume that we have saved the above config file as `classification.yaml`. We could run the following CLI command to train a Patchcore model on above dataset:
+
+```bash
+anomalib train --data classification.yaml --model anomalib.models.Patchcore --task CLASSIFICATION
+```
+
+```{note}
+As can be seen above, we also need to specify the ``task`` argument to ``CLASSIFICATION`` to explicitly tell anomalib that we want to train a classification model. This is because the default `task` is `SEGMENTATION` within `Engine`.
+```
+
+:::
+::::
+:::::
+
+### With Only Normal Images
+
+There are certain cases where we only have normal images in our dataset but
+would like to train a classification model.
+
+This could be done in two ways:
+
+- Train the model and skip the validation and test steps, as we do not have
+  abnormal images to validate and test the model on, or
+- Use the synthetic anomaly generation feature to create abnormal images from
+  normal images, and perform the validation and test steps.
+
+For now we will focus on the second approach.
+
+#### With Validation and Testing via Synthetic Anomalies
+
+We could use the synthetic anomaly generation feature again to create abnormal
```

Comment:
The description of the synthetic approach should be moved here from the segmentation section now that the order is switched